.github/workflows/build.yml

```diff
@@ -58,7 +58,7 @@ jobs:
         # Overwrite custom home dir set by GitHub Action because SSH does not respect that.
         HOME=/root
         mkdir -p ~/.ssh
-        for repo in 'cris-base' 'cris-huobi'; do
```

Comment:
We don't need them here right? Maybe simply remove all of them

Reply:
Also update the secret env name

Reply:
These files are synced across repos.

Reply:
> These files are synced across repos.

Does that make sense? cris-core should not be aware of its downstreams. If we make any changes on them or even add more, cris-core does not care.

Reply:
There's scripts/sync_infra.sh to do this automatically, for consistency and reusability across repos, not to add dependency on them.